python-base/ubuntu20.04-pyston2.2/docker/scripts/install_poetry.sh

```diff
@@ -3,23 +3,14 @@
 # shellcheck disable=SC2039
 set -exuo pipefail
 
-POETRY_URL="https://raw.githubusercontent.com/python-poetry/poetry/${POETRY_VERSION}/get-poetry.py"
+POETRY_URL="https://install.python-poetry.org"
 
 mkdir "${POETRY_HOME}"
 chown -R "${USER}":"${GROUP}" "${POETRY_HOME}"
 
-su "${USER}" -c "curl -sSL ${POETRY_URL} -o /tmp/get-poetry.py"
-su "${USER}" -c "python /tmp/get-poetry.py --version ${POETRY_VERSION}"
+su "${USER}" -c "curl -sSL ${POETRY_URL} | python - --version ${POETRY_VERSION}"
+# Create the env file manually; installers prior to 1.2.0 created this by default
+su "${USER}" -c "echo \"export PATH=\\\"${POETRY_HOME}/bin:\\\$PATH\\\"\" > \"${POETRY_HOME}/env\""
 
 chmod +x "${POETRY_HOME}"/bin/*
 bash /src/docker/scripts/configure_poetry.sh
-
```

Comment:
Why was this removed?

Reply:
There is no longer a `/usr/local/poetry/lib/poetry/_vendor/` folder, the layout changed radically due to the new way poetry is installed.